folly/Chrono.h

```diff
@@ -175,28 +175,57 @@ namespace folly {
 namespace chrono {
 
 struct coarse_steady_clock {
-  using rep = std::chrono::milliseconds::rep;
-  using period = std::chrono::milliseconds::period;
-  using duration = std::chrono::duration<rep, period>;
-  using time_point = std::chrono::time_point<coarse_steady_clock, duration>;
+  using duration = std::chrono::milliseconds;
```

Comment:
Stepping back, I'm curious why we want this to be `milliseconds` rather than alias `steady_clock::duration`.

To use `milliseconds`, we must round (`duration_cast`). Is this a necessity?

Reply:
As far as I remember, once upon a time, `struct timespec res;
    clock_getres(CLOCK_MONOTONIC_COARSE, &res);` returned an accuracy of 1 ms (on my system), and I always thought that it was. Right now, on my system, the accuracy is 4 ms (apparently due to CONFIG_HZ = 250)

Reply:
In any case, this is a less accurate watch. It would be strange if they had the same accuracy. I think most systems will have> = 1ms precision for this clock. It's okay if they are somewhere more accurate (for example, libc ++ has 1us precision on the system clock for the sake of maintaining compatibility with abi)

Reply:
I found out that the possible values of HZ are 100, 250, 300, 1000. Those the maximum accuracy at the current moment is 1ms